public/index.html

```diff
@@ -0,0 +1,17 @@
+<!DOCTYPE html>
+<html lang="">
+  <head>
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width,initial-scale=1.0">
+    <link rel="icon" href="<%= BASE_URL %>favicon.ico">
+    <title><%= htmlWebpackPlugin.options.title %></title>
```

Comment:
Is this a dynamic way to display the title of the app on the browser tab?? Very cool. I've always updated this manually.

Reply:
Yep, that's how it gets set up through the vue cli. Didn't create-react-app do that, too? I can't remember.

Reply:
Nice. With `create-react-app` it just ended up with `<title>React App</title>` or something like that and you had to manually change it to `<title>My Awesome App Name</title>`.